tests/modules/glimpse/chunk/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { GLIMPSE_CHUNK } from '../../../../modules/glimpse/chunk/main.nf' addParams( options: [args: ''] )
```

Comment:
Just curious: what does this `addParams` block do? I've never seen it before :)

Reply:
It's an old part I wrote, that should set the args parameter as an empty string.
But it's no more necessary.